src/harness/certs/generate_fake_certs.sh

```diff
@@ -123,10 +145,6 @@ openssl ca -cert sas_ca.cert -keyfile private/sas_ca.key -in admin_client.csr \
     -policy policy_anything -extensions cbsd_req_sign -config ../../../cert/openssl.cnf \
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
-
-# Generate trusted CA bundle.
-echo "\n\nGenerate 'ca' bundle"
-cat cbsd_ca.cert sas_ca.cert root_ca.cert cbsd-ecc_ca.cert sas-ecc_ca.cert root-ecc_ca.cert > ca.cert
 # Note: following server implementation, we could also put only the root_ca.cert
```

Comment:
These comments should follow the 'generate trusted CA bundle' section you moved at the bottom of this file, or remove them

Reply:
Updated   the code as per the  mentioned